### DISCUSSION TITLE
MCS - minimal code snippets

### DISCUSSION THREAD

**Discussion by omarcostahamido** [Qiskit/.github#20] [Documentation]
(Maybe too much work - Qiskit is huge, I know)
One thing I would love to see consistently in the Qiskit [reference documentation](https://qiskit.org/documentation/) pages would be the most minimal code example using each thing in the documentation. These don't have to be full examples, or even make complete sense. The sole purpose would be to show in a split second how to use or call something in the documentation - just one line or two.

**Comment by 1ucian0** [comment]
100% yes! Docstrings with examples is not something Qiskit does often enough! Let's make a list of methods of those obvious cases that need an example and they are simple enough to be few lines. They could be great for the `good first issue` tag!

**Reply by omarcostahamido** [reply]
Thank you @1ucian0 😊 
How would go about listing them? Just create individual issues? It would be cool to have a full list of all these.
Again, tying back to my [other post](https://github.com/qiskit-community/feedback/discussions/19), a full single list of the reference documentation could perhaps also make it easier to just cross-list what's been done and what is yet to do.
(btw, I'm just now trying threaded replies here on github, don't remember this in regular issue discussions..)

**Reply by javabster** [reply]
let's just start putting a list together here and maybe other people can chime in with gaps they've found! Then when we have a substantial list we can see if we can put them all in a single issue or maybe group them and create separate issues. It really depends how much there is to work on 😄

**Comment by omarcostahamido** [comment]
For the reference, [this](https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.initialize.html?highlight=initialize#qiskit.circuit.QuantumCircuit.initialize) is a beautiful example of what I meant.
In fact it could be even simpler: no need for circuit.draw() nor include the output. Whomever is looking at this already has their notebook/terminal/coding system running at the same time and can quickly see a meaningful output (in context) for themselves.

This contrasts greatly with, for example, [this](https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.unitary.html#qiskit.circuit.QuantumCircuit.unitary)

**Comment by omarcostahamido** [comment]
👋 @1ucian0 closed as resolved? :)
